karapace/utils.py

```diff
@@ -332,3 +344,33 @@ def ns_blackout(self):
 
     def blacked_out(self):
         return self.ns_blackout() or super().blacked_out()
+
+
+class DebugAccessLogger(AccessLogger):
+    """Logs access logs as DEBUG instead of INFO."""
+
+    def log(
```

Comment:
review note: This code is copied form the aiohttp library with the log level changed. So the coding style follows the one of aiohttp.

- ignoring broad except
- ignoring unidiomatic instance check

ref: https://github.com/aio-libs/aiohttp/blob/d01e257da9b37c35c68b3931026a2d918c271446/aiohttp/web_log.py#L191-L210